### DISCUSSION TITLE
❓ Saving a trained model

### DISCUSSION THREAD

**Discussion by gurbir739** [open-edge-platform/anomalib#1632] [Q&A]
How to save the model after training it with PaDim on our custom dataset

**Comment by abc-125** [comment]
If you use the default config with the custom dataset, the model should be saved in the `./results/padim/name_of_custom_dataset/run/weights/model.ckpt`.

**Reply by gurbir739** [reply]
Thanks for replying , So how can we use this model.ckpt file for getting result for detecting anomalies on test images
